### ISSUE TITLE
DELETE?Guidance on Applying WCAG 2.2 to Mobile Applications

### ISSUE THREAD

**Issue by iadawn** [w3c/wai-reviews-and-announcements#13]
Duplicate of [#6 - Guidance on Applying WCAG 2.2 to Mobile Applications](https://github.com/w3c/wai-reviews-and-announcements/issues/6)?

This is to request announcement only

- **W3C Staff Contact:** Kevin White
- **Task Force:** [Mobile Accessibility Task Force](https://www.w3.org/groups/tf/mobile-a11y-tf/)
- **Working Group:** [Accessibility Guidelines Working Group](https://www.w3.org/groups/wg/ag/)
- **Editor(s):** Jan Jaap de Groot (Abra), Jamie Herrera (Invited Expert), Joe Humbert (Invited Expert), Julian Kittelson-Aldred (Invited Expert), Jon Gibbins (Invited Expert)
- **Announcements needed?:** Yes
- **Announcement channels:** public-wai-announce, LinkedIn, Mastodon
- **Target date for announcements:**  2025-04-24 (if possible)
- **Final published document:** https://www.w3.org/TR/wcag2mobile-22/
- **Links to social media announcements:** Still huh?

Key messages for announcements:
- Work in progress
- Seeking input on the direction and approach

Summary of changes since last publication:
- First publication

Review questions:
- Does the structure of the guidance, mostly related to the Success Criteria, e.g. expand/collapse for WCAG + WCAG2ICT, followed by MATF guidance work?
- Is the wording in our guidance easy-to-read and unambiguous?
- Is there anything missing in the approach being taken?
